service-base/src/test/java/org/eclipse/hono/service/credentials/BaseCredentialsServiceTest.java

```diff
@@ -0,0 +1,187 @@
+/**
+ * Copyright (c) 2017 Bosch Software Innovations GmbH.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *    Bosch Software Innovations GmbH - initial creation
+ */
+package org.eclipse.hono.service.credentials;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.eclipse.hono.util.CredentialsConstants.FIELD_SECRETS;
+
+import org.eclipse.hono.config.ServiceConfigProperties;
+import org.eclipse.hono.util.CredentialsResult;
+import org.junit.Before;
+import org.junit.Test;
+
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Handler;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+
+/**
+ * Unit tests for BaseCredentialsService
+ *
+ */
+public class BaseCredentialsServiceTest {
+
+    private BaseCredentialsService<ServiceConfigProperties> service;
+
+    private static final String TEST_FIELD = "dummy_test";
```

Comment:
IMHO to only call it "test" is enough. To use "dummy" as well looks doubled to me.

Reply:
will be changed